app/controllers/reviews_controller.rb

```diff
@@ -1,4 +1,6 @@
 class ReviewsController < ApplicationController
+  before_action :authenticate_user!, only: [:new, :create, :edit, :update, :destroy]
```

Comment:
May help you resolve hound issue by using `except:` (which is the inverse meaning of `only:`), as in example below:

```ruby
before_action :authenticate_user!, except: [:index, :show]
```